src/Simple.OData.Client.Core/Adapter/BatchWriterBase.cs

```diff
@@ -27,9 +28,12 @@ protected BatchWriterBase(ISession session, IDictionary<object, IDictionary<stri
         public abstract Task<HttpRequestMessage> EndBatchAsync();
 
         public async Task<ODataRequest> CreateBatchRequestAsync(
-            IODataClient client, IList<Func<IODataClient, Task>> actions, IList<int> responseIndexes)
+            IODataClient client, IList<Func<IODataClient, Task>> actions, 
```

Comment:
Is there a reason we need to remove `responseIndexes` from a public interface? That's a potentially big breaking change for some folks.